### PR TITLE
docs: edit project-structure.md

### PR DESCRIPTION
**Pull request by dejmedus**
PROJECT-STRUCTURE.md for issue #48

* Caught a few minor typos.
* Most comments started with a lowercase letter and ended without a period, so I changed all the comments to follow that structure.
* Removed some instances of "this is", "all", or "contains" to keep comments concise.

### CODE REVIEW

File: .docs/PROJECT-STRUCTURE.md [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
...%2BC5Zil8nam8i3%5E6local-c5XSCSSZil8nam8i3%256%3FFJ%5B.tsx*6%5B2%2F*%20%401Xthi3singl8scs3fil8i39all%20local%20c53iKth8Page*%20PageName1%2BContain3th8OC5*%20%3F.pgFre-%3C3OC5%20a3a%20%5C%27defaulM%3C%5C%27*%20hooksB7local%3E%3D*%402%2F*_app.pg.t37WebApp%223rooMApp%20c5*_%7C.pgFref-%20https%3A%2F%2Fnextjs.org%2Fdocs%2Fadvanced-features%2Fcustom-%7C*_gH.scss37gH%20styles.%20%7BRarely%20used%7D*404.pgFouqcustom%20404%20erroqpageQpublicB%23Do%20noMpuMoqediManyZile3insid8thisQtheme*%5D%2BouqThem8Provideqc5*themeFgenerate3ouqtheme3foqc5s.%20StrictlyZollow%20STYLE-GUIDE.md%20wheKediting%20this.*%3FFre-%3C3everythingZrom%20%5D.tsxQtypesB%23contian3gH%20TypeScripMtypesQ%24iesQ.gitignoreQpackage.json%27)~version!%271%27)*Q63s%205omponent6%20%207%23contain38e%209used%20by%20B%2F%20F.t3%23HlobalJthi3fil8re-%3C3th8ReacMC5Zrom%20Kn%20Mt%20OPageName1%20Q%5CnV%5C%5C*X.module.scs3%23Z%20fqr%20%24utilsB%23miscellaneou3utilit%25iKkebab-case*6%263%7B9mor8thaKon8pages%7DV%2B.tsx%20%23%3B%20endpoin%3Cexport%3D3%7B9Opag8only%7D%3E%20custom%20hook%3Findex%40page-name%5BLocalC5%5DWPThemeProvider%5EiKPascalCase*6%7Cdocument%7Fasset3used%20%01%7F%7C%5E%5D%5B%40%3F%3E%3D%3C%3B%2B%26%25%24qZXVQOMKJHFB987653*>)
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:57]: Can you update this Tree too? Go to that website (click the url). Update that tree in that website. Generate new URL for that updated tree. Put/update the new link here

**Review by ashuvssut — CHANGES_REQUESTED** [review]
Rest looks LGTM!

File: .docs/PROJECT-STRUCTURE.md [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
-│   │   ├── route1/ #this is route1 endpoint
+│   ├── api/ #Nextjs API endpoints
+│   │   ├── route1/ #route1 endpoint
 │   │   │   └── index.pg.ts #route1 endpoint logic
-│   │   ├── utils/ #miscellaneous utilites used by endpoints
-│   │   └── index.pg.ts #contains root (/) endpoint handler
+│   │   ├── utils/ #miscellaneous utilities used by endpoints
+│   │   └── index.pg.ts #root (/) endpoint handler
 │   ├── page-name1/
-│   │   ├── components/ #contains local components (used by PageName1 page only)
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:31]: `PageName1` refers to the components. `page-name1` is the page-route/folder-name. So intead you can write `(used by PageName1 page component only`

File: .docs/PROJECT-STRUCTURE.md [hunk 3]
```diff
@@ -1,57 +1,57 @@
...
+│   │   │   │   ├── LocalComponent1.tsx #component file names are in PascalCase
+│   │   │   │   ├── local-component.module.scss #SCSS file names are in kebab-case
+│   │   │   │   └── index.ts #re-exports the React component from LocalComponent.tsx
 │   │   │   └── LocalComponent2/
-│   │   ├── page-name1.module.scss #this single scss file is used by all local components in the Page
-│   │   ├── PageName1.tsx #Contains the PageName1 Component
-│   │   ├── index.pg.ts #re-exports PageName1 Component as a "default export"
-│   │   └── hooks/ #contains local custom hooks (used by PageName1 page only)
+│   │   ├── page-name1.module.scss #this single SCSS file is used by all the local components in page-name1/
+│   │   ├── PageName1.tsx #page-name1/ component
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:38]: same correction. Its PageName1 compoenent (not page-name1/ component)

File: .docs/PROJECT-STRUCTURE.md [hunk 4]
```diff
@@ -1,57 +1,57 @@
...
+│   │   ├── index.pg.ts #re-exports page-name1/ component (PageName1.tsx) as a "default export"
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:39]: Revert this line change

File: .docs/PROJECT-STRUCTURE.md [hunk 5]
```diff
@@ -1,57 +1,57 @@
...
+│   │   └── hooks/ #local custom hooks (used by page-name1/ only)
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:40]: same change... PageName1 component (not page-name1/ component)

File: .docs/PROJECT-STRUCTURE.md [hunk 6]
```diff
@@ -1,57 +1,57 @@
...
 │   ├── page-name2/
-│   ├── _app.pg.ts #contains WebApp's root App component
+│   ├── _app.pg.ts #walking-pal-web's root App component
 │   ├── _document.pg.ts #ref- https://nextjs.org/docs/advanced-features/custom-document
-│   ├── _global.scsss #contains global styles. (Rarely used)
-│   └── 404.pg.ts #our custom 404 error page
-├── public/ #Do not put or edit any files inside this
+│   ├── _global.scsss #rarely used global styles
```
Comment (ashuvssut) [inline .docs/PROJECT-STRUCTURE.md:44]: revert this line change change too.

File: .docs/PROJECT-STRUCTURE.md (hunk 1 above)
Comment (dejmedus) [reply .docs/PROJECT-STRUCTURE.md]: I updated the link

File: .docs/PROJECT-STRUCTURE.md (hunk 2 above)
Comment (dejmedus) [reply .docs/PROJECT-STRUCTURE.md:31]: I changed the PageName1 section back to normal

File: .docs/PROJECT-STRUCTURE.md (hunk 6 above)
Comment (dejmedus) [reply .docs/PROJECT-STRUCTURE.md]: Global styles line is changed back also

**Comment by ashuvssut** [comment]
LGTM! Nice edits!

**Pull request merged**